legal-api/pyproject.toml

```diff
@@ -34,7 +34,8 @@ requests = "^2.31.0"
 strict-rfc3339 = "^0.7"
 minio = "^7.1.15"
 html-sanitizer = "^1.9.3"
-registry-schemas = {git = "https://github.com/bcgov/business-schemas.git", rev = "2.18.10"}
+#registry-schemas = {git = "https://github.com/bcgov/business-schemas.git", rev = "2.18.10"}
+registry-schemas = {git = "https://github.com/argush3/business-schemas.git", branch = "dev_legal_name_changes"}
```

Comment:
Will reference `{git = "https://github.com/bcgov/business-schemas.git", rev = "2.18.10"}
registry-schemas = {git = "https://github.com/bcgov/business-schemas.git", branch = "dev_legal_name_changes"}` once https://github.com/bcgov/business-schemas/pull/140 is merged